haskell/haskell.bzl

```diff
@@ -163,6 +196,11 @@ Example:
       deps = ["//lib:some_lib"]
   )
   ```
+
+Every `haskell_binary` target also defines an optional REPL target that is
+not built by default, but can be built on request. The name of the REPL
+target is the same as the name of binary with `"-repl"` added at the end.
+For example, the target above also defines `main-repl`.
```

Comment:
How to load a library or binary into a REPL should be documented (use `bazel run --direct_run`?). Note that in future versions of Bazel, `--direct_run` *might* become the default. See https://github.com/bazelbuild/bazel/commit/1fafc6fd3ac87d2963d08a6b7ddd076edfa2fb94.

Reply:
I added something about this.